src/components/UserForm/FormInput/FormInput.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { string } from 'prop-types';
+
+const FormInput = ({ labelText, type, name, placeholder }) => {
```

Comment:
If FormInput is also used in OrderForm, why it stays under UserForm and not as a standalone component?